specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```diff
@@ -378,63 +674,360 @@
+      },
+      "required": [
+        "analysisInput",
+        "tasks"
+      ]
+    },
+    "AnalyzeConversationJobsInput": {
+      "properties": {
+        "displayName": {
+          "description": "Optional display name for the analysis job.",
+          "type": "string"
+        },
+        "analysisInput": {
+          "$ref": "#/definitions/MultiLanguageConversationAnalysisInput"
+        },
+        "tasks": {
+          "description": "The set of tasks to execute on the input conversation.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AnalyzeConversationLROTask"
+          }
+        }
+      },
+      "required": [
+        "analysisInput",
+        "tasks"
+      ]
+    },
+    "AnalyzeConversationLROTask": {
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationLROTaskKind"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/TaskIdentifier"
+        }
+      ]
+    },
+    "AnalyzeConversationLROTaskKind": {
+      "type": "string",
+      "description": "Enumeration of supported Conversation Analysis LRO tasks.",
+      "enum": [
+        "TranscriptionPii",
+        "IssueResolutionSummarization"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationLROTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationLROResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported Conversation Analysis task results.",
+      "enum": [
+        "TranscriptionPiiResults",
+        "IssueResolutionSummarizationResults"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationLROResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTranscriptionPiiLROTask": {
+      "type": "object",
+      "description": "Task definition for a PII redaction in transcriptions.",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/TranscriptionPiiTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "TranscriptionPii"
+    },
+    "TranscriptionPiiConversationItemResult": {
+      "type": "object",
+      "required": [
+        "id",
+        "redactedContent"
+      ],
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "redactedContent": {
+          "type": "object",
+          "$ref": "#/definitions/RedactedTranscriptContent"
+        },
+        "entities": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Entity"
+          }
+        }
+      }
+    },
+    "AudioTiming": {
+      "type": "object",
+      "properties": {
+        "offset": {
+          "description": "Offset from start of speech audio, in ticks.",
```

Comment:
nit: Might be good to describe a "tick" since its not very well known measurement. See [here](https://docs.microsoft.com/en-us/dotnet/api/system.timespan.ticks?view=net-6.0#remarks).

Reply:
added